ci-operator/step-registry/ignition/bastionhost/ignition-bastionhost-commands.sh

```diff
@@ -85,6 +85,11 @@ EOF
 cat > ${workdir}/squid.conf << EOF
 auth_param basic program /usr/lib64/squid/basic_ncsa_auth /etc/squid/passwords
 auth_param basic realm proxy
+#prfsr90
```

Comment:
I suggest adding a switch to enable a particular configuration for prfsr90 so that the common test is not affected.

Reply:
I am not going to merge this pr yet.  once I finish this poc for one of the customer-critical ticket, I will either disable it or think of adding as a regression case if I get a review on it. thank you

Reply:
OK